tools/automation/release/check_version.py

```diff
@@ -20,7 +20,7 @@ def is_this_a_good_version_number(string: str) -> Optional[str]:
     if v.dev:
         return "No development releases on PyPI. What are you even thinking?"
 
-    if v.is_prerelease and v.pre[0] != "b":
+    if v.is_prerelease and (v.pre is None or v.pre[0] != "b"):
```

Comment:
This is what I had earlier, but changed it because we hadn't added the required annotations in packaging. Maybe it works now? Or maybe we need to tell mypy to include packaging at runtime?

```suggestion
    if v.pre and v.pre[0] != "b":
```

Reply:
Yes, that seems to work on my machine.  I've updated the patch accordingly.